src/node/signatures.h

```diff
@@ -32,6 +32,8 @@ namespace ccf
       commit(0)
     {}
 
+    Signature(crypto::Sha256Hash root_) : root(root_) {}
```

Comment:
What's the use case for this? Saving signatures with uninitialised term, index etc does not sound at all.

Reply:
For PBFT all of this information is stored in the pre-prepare. It is the signature hash that we need to store in the signatures table for the members to use when they `ack`. I don't think we use the other signature information anywhere else, and since the signature table is not persisted to the ledger and the pre-prepare that contains all of that info is, I didn't think we should duplicate the data

Reply:
Can we still zero it though?

Reply:
Zero'd